_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,465 @@
+            <input class="field-input input-text" type="text" name="full_name_default" id="full_name_default" placeholder="e.g. Jane Doe">
+        </div>
+        <div class="field">
+            <label class="field-label" for="email_address_default">Email address (default, disabled text field)</label>
+            <input class="field-input input-text" type="email" name="email_address_default" id="email_address_default" placeholder="e.g. jane@email.com" disabled>
+        </div>
+        <div class="field">
+            <label class="field-label field-required" for="email_address_default_helper">Email address (default, required text field with helper text)</label>
+            <input class="field-input input-text" type="email" name="email_address_default_helper" id="email_address_default_helper" placeholder="e.g. jane@email.com" aria-describedby="email_address_default_helper" required>
+            <div class="field-hint" id="email_address_default_helper">
+                <p>Your email address will never be shared. We promise.</p>
+            </div>
+        </div>
+        <div class="field">
+            <label class="field-label field-required" for="email_address_default_err_helper">Email address (default, has error)</label>
+            <input class="field-input input-text has-error" type="email" name="email_address_default_err_helper" id="email_address_default_err_helper" placeholder="e.g. jane@email.com" aria-describedby="email_address_default_err_helper" required>
+            <div class="field-hint" id="email_address_default_err_helper">
+                <p>Your email address will never be shared. We promise.</p>
+            </div>
+        </div>
+        <div class="field">
+            <label class="field-label field-required" for="email_address_default_suc_helper">Email address (default, has success)</label>
+            <input class="field-input input-text has-success" type="email" name="email_address_default_suc_helper" id="email_address_default_suc_helper" placeholder="e.g. jane@email.com" aria-describedby="email_address_default_suc_helper" required>
+            <div class="field-hint" id="email_address_default_suc_helper">
+                <p>Your email address will never be shared. We promise.</p>
+            </div>
+        </div>
+        <div class="field">
+            <label class="field-label" for="full_name_simple">Your name (simple text field)</label>
+            <input class="field-input input-text input-alt" type="text" name="full_name_simple" id="full_name_simple" placeholder="e.g. Jane Doe" required>
+        </div>
+        <div class="field">
+            <label class="field-label" for="full_name_simple_err">Your name (simple text field, error)</label>
+            <input class="field-input input-text input-alt has-error" type="text" name="full_name_simple_err" id="full_name_simple_err" placeholder="e.g. Jane Doe" required>
```

Comment:
NIT: since this is an error state, should it have a value entered (to mirror the real world case of user input needed for failed validation)?